phoenix-core/src/it/java/org/apache/phoenix/end2end/MaxLookbackIT.java

```diff
@@ -60,7 +60,7 @@
 
 @NeedsOwnMiniClusterTest
 public class MaxLookbackIT extends BaseUniqueNamesOwnClusterIT {
-    private static final Log LOG = LogFactory.getLog(MaxLookbackIT.class);
+    public static final Logger LOG = LoggerFactory.getLogger(MaxLookbackIT.class);
```

Comment:
nit: keep it private? applicable to all places.

Reply:
Of course. Thanks for catching this copy-paste error.

Reply:
Done